README.md

```diff
@@ -183,14 +180,19 @@ Each team will have team leaders who are responsible for keeping track of each t
   - Ensures the task list stays up to date
 
 ## Stand Up Meetings
-The Project Manager for your team will determine the timing for all Stand Up meetings. Because PMs are managing multiple projects at once this time will be different for each team, and may change from day to day. The meeting schedule will be communicated to the Stand Up Leader as soon as it is determined.
+The Stand Up Leader should determine the daily time for your stand up meeting with the team. Once you come up with a time, confirm with your PM that this time will work for them.
 
-At the end of each day, your team's assigned Project Manager will review the Trello board for all tasks discussed during that day's Stand Up meeting.
+At the end of each day, your team's assigned Project Manager will review the Trello board to ensure it captures the updates that your team has made throughout the day.
 
-## Weekly Demos
+## Interim Demo
 In a real world work environment, a team's success is measured by their product as opposed to each individual's contribution.
 
-Each team will present their progress and respond to questions from their Project Manager each Friday. Every team member will participate in these demos; the PM will ask specific questions regarding
+Each team will present their progress and respond to questions from their Project Manager on the first Friday. Every team member will participate in these demos; the PM will ask specific questions regarding
 1. The team's progress and plan for completing the project
 1. The technical decisions and implementation
-1. Every team member's understanding of the underlying technical structures.
+1. Every team member's understanding of the underlying technical structures
+
+## Final Presentation
```

Comment:
I suggest a time constraint on the presentation., 10 min?

Reply:
I'll make it an arbitrary 7 minutes to keep in well under 10 to allow for transition time and a one hour overall presentation for a class.